stafihub/chain.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "stafihub",
+  "chain_id": "stafihub-1",
+  "website": "https://stafihub.io/",
+  "pretty_name": "StaFi Hub",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "stafi",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md